src/ripple/protocol/tokens.h

```diff
@@ -76,6 +76,18 @@ std::string
 base58EncodeToken (std::uint8_t type,
     void const* token, std::size_t size);
 
+/*  Base-58 encode a Bitcoin Token
+ *
+ *  provided here for symmetry, but should never be needed
+ *  except for testing.
+ *
+ *  @see base58EncodeToken for format description.
```

Comment:
Since you're referring to it, could you update the `@param`s for `base58EncodeToken` above?